lib/auth.coffee

```diff
@@ -215,6 +216,43 @@ getAuth = (deps, opts) ->
 		.catchReturn(false)
 		.asCallback(callback)
 
+	###*
+	# @summary Creates a new API key for user
```

Comment:
I think that we need a `the` before the `user`.
How about `Creates a new user API key`?
cc: @pimterry